README.md

```diff
@@ -1,44 +1,18 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Rederly Frontend
 
-## Available Scripts
+![Version](https://img.shields.io/github/v/release/rederly/frontend?style=plastic)
```

Comment:
How is this getting it's status, is this hooked up to the github action?

Reply:
This is getting it from our Releases tab.